faker_edu/__init__.py

```diff
@@ -172,3 +201,9 @@ def faculty_title(self):
         position = self.faculty_position()
         department = self.department_name()
         return position+" of "+department
+
+    def sport(self):
+        genders = ["Men's", "Women's"]
```

Comment:
Not sure we always want sports to be gendered. For example men's and women's college will often simply refer to their team by the sport name and other similar things.  Might be better to add the gender either at random or include a parameter to control the behavior (something like a boolean `includeGender`)